### PR TITLE
Implement prettier and module resolution

### PR DESCRIPTION
**Pull request by edwincapel**
What this PR does

- Implement prettier for better code formatting
- Import order config
- Module resolution

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -98,6 +100,12 @@
...
     /* Completeness */
     // "skipDefaultLibCheck": true,                      /* Skip type checking .d.ts files that are included with TypeScript. */
-    "skipLibCheck": true                                 /* Skip type checking all .d.ts files. */
+    "skipLibCheck": true /* Skip type checking all .d.ts files. */,
+    "paths": {
+      "types/*": ["src/types/*"],
+      "decorators/*": ["src/decorators/*"],
+      "plugins/*": ["src/plugins/*"]
+    },
+    "baseUrl": "./"
```
Comment (edwincapel) [inline tsconfig.json:109]: Module resolution

File: .prettierrc.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "semi": false,
+  "tabWidth": 2,
+  "singleQuote": true,
+  "trailingComma": "es5",
+  "printWidth": 100,
+  "importOrder": ["<THIRD_PARTY_MODULES>", "types/.*$"],
+  "importOrderSeparation": false
+}
```
Comment (edwincapel) [inline .prettierrc.json:9]: prettier config

File: tsconfig.json (hunk 1 above)
Comment (jeroenouw) [reply tsconfig.json:109]: Nice!

File: src/decorators/db.ts [hunk 3]
```diff
@@ -1,11 +1,15 @@
-import Elysia from "elysia";
-import { DbAction } from "../types/database";
-import Database from "bun:sqlite";
+import Elysia from 'elysia'
+import { DbAction } from 'types/database'
+import Database from 'bun:sqlite'
```
Comment (jeroenouw) [inline src/decorators/db.ts:2]: I would guessed that this import would be above the types now, with prettier.

Comment (edwincapel) [reply src/decorators/db.ts:2]: reordered imports

**Pull request merged**